test/test_sync.cpp

```diff
@@ -5904,8 +5904,6 @@ TEST(Sync_ClientErrorHandler)
 }
 
 
-#ifndef REALM_PLATFORM_WIN32
```

Comment:
This define got dropped when moving these to the core repo and is never actually set.